tap_linkedin_ads/streams.py

```diff
@@ -118,6 +119,10 @@ class Accounts(LinkedInAdsStream):
         ),
     ).to_dict()
 
+    @property
+    def url_base(self) -> str:
+        return "https://api.linkedin.com/rest/"
```

Comment:
Hey Edgar, in the streams updated in the 202305 update, we have to update the base_url to be `"https://api.linkedin.com/rest/adAccounts/"`. However, changing the url_base in streams.py causes a `Cannot override writeable attribute ` error because it was set in client.py. Thus, we decided to move the url_base function to each stream to avoid the error.

Reply:
Ok, no biggie then 👍